mod_reforged/hooks/ai/tactical/behavior.nut

```diff
@@ -0,0 +1,12 @@
+::mods_hookBaseClass("ai/tactical/behavior", function(o)
+{
+    o = o[o.SuperName];
+
+    local queryTargetValue = o.queryTargetValue;
+    o.queryTargetValue = function( _entity, _target, _skill = null )
+    {
+        // Currently Mortars are near invincible and not meant to be killed. But AI still targets them sometimes. This line should make it so they are basically never targeted.
+        if(_entity.getType() == ::Const.EntityType.Mortar) return 0.01;
```

Comment:
I see that the original `queryTargetValue` function takes into account the target's HP. Do you have an estimate of what the return value is for mortars? How big is it compared to this new `0.01`?

Reply:
I just looked into the queryTargetValue and while I don't understand everything the following line should give some sort of answer:
`score = score + hitpoints * this.getProperties().TargetPriorityHitpointsMult;`
`hitpoints` is the fraction of expectedDamage/remainingHitpoints
`TargetPriorityHitpointsMult` is some multiplier value, between 0.15 and 0.50 across the board.

But apart from this there are about 20 other lines of code that all add to `score`. For Mortar most of them will likely not fire.
The most notable one that triggers is
`score = score + hitchance / 100.0 * this.getProperties().TargetPriorityHitchanceMult * this.Const.AI.Behavior.GlobalHitchanceValueMult;`
Every AI except Alps will consider Mortars an interesting target just because they have a realistic chance of hitting it. And this is basically always greater than 0.01.